### PR TITLE
CMDCT-3104: [PDF] SAR General Information Page + Metadata Table

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Creating and styling the PDF for the `SAR General Information` page (and its internal sections), as well as pulling the metadata for the PDF.

### Related ticket(s)

CMDCT-3104

---
### How to test

There's the _old_ way:
- Log in as a state user
- Create and submit a Work Plan
- Log in as an admin user
- Approve the Work Plan
- Log back in as state user
- - Create a new SAR
- Fill in the `General Information` page (the first page of this form)
- Navigate to `Review & submit` and click on `Review PDF`

And then there's the _coooool_ way:
- Use @gmrabian and @keeysnc 's handy dandy `MFP Postman Collection` to create, fill, submit, AND approve a WP 😎 
- Create a new SAR
- Fill in the `General Information` page (the first page of this form)
- Navigate to `Review & submit` and click on `Review PDF`

The metadata tables at the top of the PDF should have all the correct data populating and you should see the `General Information` section + its subsections with the appropriate data.

**Note:** I left the red border around these tables to avoid any confusion on which sections are done, but I could remove it if needed!

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by ailZhou — DISMISSED** [review]
looks good! tested locally, all the entered text are displaying in the pdf.

minor issue is that the table response row text isn't lining up with the header but styling can be a different pr.
Screenshot 2024-01-10 at 4 40 22 PM

File: services/ui-src/src/verbiage/pages/sar/sar-export.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ export default {
   reportPage: {
     heading: "Managed Care Program Annual Report (SAR) for ",
```
[the remark this answers is not in the document]
Comment (karla-vm) [reply services/ui-src/src/verbiage/pages/sar/sar-export.ts:16]: Great call, I think this was brought in from MCR verbatim. Fixing it!

**Comment by karla-vm** [comment]
> looks good! tested locally, all the entered text are displaying in the pdf.
> 
> minor issue is that the table response row text isn't lining up with the header but styling can be a different pr. Screenshot 2024-01-10 at 4 40 22 PM

just fixed that! thanks, Casie 🏆

**Review by cassandradanger — APPROVED** [review]
looks great!!

**Review by cassandradanger — APPROVED** [review]

**Pull request merged**